framework/org.eclipse.concierge/test/org/eclipse/concierge/BundleGetDataFileTest.java

```diff
@@ -121,8 +132,8 @@ public void testGetDataFileWithFiles3Level() throws Exception {
 
 		File f1 = bundleUnderTest.getDataFile("a/b/file.txt");
```

Comment:
I also did not run the code in Windows: does this work with `/` as separator here?
Maybe there are more places to check on file separator.

Reply:
It works on windows. `File` accept path with `/` and `\` on each platform.
So, `a/b/file.txt` is works on Windows.

But, some test cases does not work because there are compared in String.